ledger_device_sdk/src/nbgl.rs

```diff
@@ -305,16 +288,23 @@ impl<'a> NbglHomeAndSettings<'a> {
             }
 
             loop {
-                if self.glyph.is_some() {
-                    self.icon = &self.glyph.unwrap().into() as *const nbgl_icon_details_t;
-                }
+                let info_list: nbgl_contentInfoList_t = nbgl_contentInfoList_t {
```

Comment:
Same remark as with the icon, I had issues when passing a local content variable to the NBGL API. Please verify the behavior of the app in functional tests with this change.

Reply:
Ok I can't reproduce the bugs I saw before, maybe I was not declaring the variables in the loop and it was causing issues.